lifecycle_msgs/CMakeLists.txt

```diff
@@ -0,0 +1,30 @@
+cmake_minimum_required(VERSION 3.5)
+
+project(lifecycle_msgs)
+
+if(NOT WIN32)
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 -Wall -Wextra")
+endif()
+
+find_package(ament_cmake REQUIRED)
+find_package(rosidl_default_generators REQUIRED)
+find_package(std_msgs REQUIRED)
+
+set(msg_files
+  "msg/Transition.msg"
+)
+set(srv_files
+  "srv/GetState.srv"
+  "srv/ChangeState.srv"
+)
+
+rosidl_generate_interfaces(${PROJECT_NAME}
+  ${msg_files}
+  ${srv_files}
+  DEPENDENCIES std_msgs
```

Comment:
I don't think that you need to depend on `std_msgs` if you are using only primitive types (dpeendency can be removed from `package.xml` too)